scripts/generate_errors.pl

```diff
@@ -46,7 +46,14 @@
 
 $/ = $line_separator;
 
-open(GREP, "grep \"define MBEDTLS_ERR_\" $include_dir/* |") || die("Failure when calling grep: $!");
+my @files = <$include_dir/*>;
+my @matches;
+foreach my $file (@files) {
+    open(FILE, "$file");
+    my @grep_res = grep(/define MBEDTLS_ERR_/, <FILE>);
```

Comment:
I agree with @gilles-peskine-arm. If we're going to change the script, we should also take the opportunity to make the pattern matching a little more robust.